Externals/wxWidgets3/include/wx/bitmap.h

```diff
@@ -305,13 +302,7 @@ wxBitmap::
 #endif
 ConvertToDisabled(unsigned char brightness) const
 {
-    // XXX comex: scale support
-    wxImage disabledImage = ConvertToImage().ConvertToDisabled(brightness);
-    #ifdef __APPLE__
-    return wxBitmap(disabledImage, -1, GetScaleFactor());
-    #else
-    return disabledImage;
-    #endif
+    return ConvertToImage().ConvertToDisabled(brightness);
```

Comment:
If I recall, handling for 2x is supported much better in the master version (this scaling should be done in the wxcocoa specific code now). I can look into this if you'd like.